src/udpreplay.cpp

```diff
@@ -182,21 +200,46 @@ int main(int argc, char *argv[]) {
                                                   ip->ip_hl * 4);
       if (interval != -1) {
         // Use constant packet rate
-        usleep(interval * 1000);
+        deadline.tv_sec += interval / 1000L;
+        deadline.tv_nsec += interval * 1000000L;
```

Comment:
This code would produce de-normalized deadline (i.e. tv_nsec  > 1000000L) for interval > 1.0.

Reply:
yeah should be `%`